api/models/user.model.js

```diff
@@ -1,7 +1,18 @@
 const mongoose = require('mongoose');
 const crypto = require('crypto');
 
-const USER_ROLES = ['superadmin', 'admin', 'mod', 'member'];
+const USER_ROLES = [
```

Comment:
I think we need to consider possibly making two roles - one for admin, superadmin, mod, member, and one for the roles from the RAI Collab tool. If you look in the database there is a separate collection of 'roles' that already contains several of the new ones. We should look at leveraging that collection and having both a `USER_ROLE` and another role (not sure what we want to call that).